Jenkinsfile

```diff
@@ -374,6 +374,7 @@ stage('Default build') {
 		return
 	}
 	node(NODE_PATTERN_BASE) {
+		cleanWs()
```

Comment:
Hum, does that mean that we will need to download again all the dependencies before building? Wondering if we could maybe only clean the artifacts we built?

Reply:
I implemented a more selective cleanup and it seems to work, as this build confirms: http://ci.hibernate.org/blue/organizations/jenkins/hibernate-search-yoann/detail/ci-test-this-please/31/pipeline/68 (see how the apache artifacts are there all along, but the Hibernate Search artifacts are deleted before the build).
I had to also avoid cleaning up the `.git` directory, because deleting it would cause the next `checkout scm` command to clean up the whole workspace.

It turns out the workspace is only shared among all the builds **for the same branch/PR**. So t's really unlikely for the in-workspace cache to be of any use across builds. But it'll be there...